README.md

```diff
@@ -152,6 +163,24 @@ VALID_ISSUERS=
 
 Once the self service app is started, the aggregator api will work as well
 
+## NPM Commands
+
+| Command               | Description                                                                                                  |
+| --------------------- | ------------------------------------------------------------------------------------------------------------ |
+| `npm start`           | Run server which serves production ready build from `dist` folder                                            |
+| `npm start-local`     | Run app locally in the `development` mode and `dev` config (calls on `npm run dev`)                                           |
+| `npm run dev`         | Run app in the `development` mode and `dev` config                                                           |
+| `npm run dev-https`   | Run app in the `development` mode and `dev` config using HTTPS protocol                |
+| `npm run prod`        | Run app in the `development` mode and `prod` config                                                          |
+| `npm run build`       | Build app for production and puts files to the `dist` folder, default to `development` mode and `dev` config |
+| `npm run analyze`     | Analyze dependencies sizes and opens report in the browser                                                   |
+| `npm run lint`        | Check code for lint errors                                                                                   |
+| `npm run format`      | Format code using prettier                                                                                   |
+| `npm run test`        | Run unit tests                                                                                               |
+| `npm run watch-tests` | Watch for file changes and run unit tests on changes                                                         |
+| `npm run coverage`    | Generate test code coverage report                                                                           |
+| `npm run mock-api`    | Start the mock api which mocks Recruit api                                                                   |
+
 ## Linting
 
 ### Rules
```

Comment:
@brooketopcoder Why TSLint is installed globally? Any reason behind this?

Reply:
@hentrymartin We had to install it globally so that we could run `tslint` from the command line, outside of running it via npm in the project. If memory services, I think we were also running into issues just getting npm to find and run it, but that was a month or more ago so my memory isn't great on that.

Reply:
ok thanks @testflyjets

Reply:
@hentrymartin good catch. tslint doesn't technically need to be installed globally, but it does need to be installed for each project that uses it. so it just seemed easier to install globally.

Reply:
ok thanks for the clarification 👍🏻